lambda-layers/terraform/Makefile

```diff
@@ -0,0 +1,52 @@
+export AWS_ACCESS_KEY_ID ?= test
+export AWS_SECRET_ACCESS_KEY ?= test
+export AWS_DEFAULT_REGION = us-east-1
+SHELL := /bin/bash
+PYTHON_BIN ?= $(shell which python3 || which python)
+
+usage:       ## Show this help
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+start:
+	localstack start -d
+
+install:    ## Install dependencies
+	@which localstack || pip install localstack
+	@which awslocal || pip install awscli-local
+	@which terraform || (\
+	echo 'Terraform was not found, installing locally' && \
+	wget https://releases.hashicorp.com/terraform/1.0.8/terraform_1.0.8_linux_amd64.zip && \
+	unzip terraform_*.zip && \
+	rm terraform_*.zip)
+	@test -e .venv || ($(PYTHON_BIN) -m venv .venv; source .venv/bin/activate; pip3 install terraform-local;)
+
+package:
+	source .venv/bin/activate; pip install \
```

Comment:
A comment pointing to the AWS packaging docs for Python would be helpful here: https://docs.aws.amazon.com/lambda/latest/dg/python-package.html
In particular, mention the limitation that dependencies with native dependencies require further configuration and should ideally be build using Docker.